sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -308,15 +313,20 @@ private static String generatePageName(Class<?> activityClass) {
      */
     private synchronized AnalyticsTransmissionTarget getInstanceTransmissionTarget(String transmissionTargetToken) {
         if (transmissionTargetToken == null || transmissionTargetToken.isEmpty()) {
-            AppCenterLog.error(LOG_TAG, "Transmission target may not be null or empty.");
+            AppCenterLog.error(LOG_TAG, "Transmission target token may not be null or empty.");
+            return null;
+        } else if (mContext == null) {
+
+            // Context and channel have the same lifecycle so we only have to check one.
```

Comment:
We use `/*` comments when its not generated by Android Studio to ignore warnings.